social/pushover/57-pushover.js

```diff
@@ -35,6 +36,7 @@ module.exports = function(RED) {
             var sound = this.sound || msg.sound || null;
             var url = this.url || msg.url || null;
             var url_title = this.url_title || msg.url_title || null;
+            var html = this.html || msg.html || false;
```

Comment:
If the user sets the flag to false then it should be honoured. This won't do that - as the this.html will evaluate false and then skip to the next option (msg.html)... so msg.html could always override it.

Reply:
Good point. I removed the option to toggle HTML mode from the incoming msg.
Now that I've given it some more thought, I can't think of a reasonable case where you would need to make this dynamic and change based on the `msg` anyway.